site/src/pages/apis/global/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import ApiReference from '../../../components/ApiReference';
+import SpecVersionDropdown from '../../../components/SpecVersionDropdown';
+
+export default function ApiReferencePage() {
+    const {siteConfig} = useDocusaurusContext();
+    const options = [
+        {title: "V1", link: "/apis/global"},
```

Comment:
@ckoegel The old link will be gone. Yesterday, Hannah Magnan changed the link on the main Bandwidth site to point to https://dev.bandwidth.com/apis/bwi/. You have to contact her to change it again to the new link.

Reply:
we could also just set up a redirect in the lambda